client/src/components/FiveBooks.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import $ from 'jquery';
+
+class FiveBooks extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      titles: [],
+    };
+
+    this.onHover = this.onHover.bind(this);
+  }
+
+  componentDidUpdate() {
+    if (this.props.books !== undefined && this.state.titles.length === 0) {
```

Comment:
This may be extraneous - unless you have something in mind I'm not thinking of, I don't think you need to move the book titles from `props` to `state`.

Reply:
Yes, I was running into a problem getting the 'child' component to render the props, so I was trying different approaches to get it to work, but I fully agree with your assessment; I will change this in a future PR.